tests/formik/BlockNavigation.test.jsx

```diff
@@ -49,36 +49,36 @@ describe("formik/BlockNavigation", () => {
     expect(screen.queryByText(/test page/i)).not.toBeInTheDocument();
   });
 
-  it("should allow navigation when form is empty", () => {
+  it("should allow navigation when form is empty", async () => {
     render(<TestBlockNavigation />);
 
-    userEvent.click(screen.getByRole("link"));
+    await userEvent.click(screen.getByRole("link"));
     expect(screen.getByText(/test page/i)).toBeInTheDocument();
   });
 
   it("should not allow navigation when form isn't empty", async () => {
     render(<TestBlockNavigation />);
 
     const input = screen.getByRole("textbox");
-    await waitFor(() => {
-      userEvent.type(input, "test");
-      userEvent.click(screen.getByRole("link"));
+    await waitFor(async () => {
+      await userEvent.type(input, "test");
+      await userEvent.click(screen.getByRole("link"));
     });
```

Comment:
This `waitFor` might not be required, could you check?

Reply:
You are right @ajmaln. I have updated it accordingly.